helm/values.yaml

```diff
@@ -542,8 +542,7 @@ global:
             connectionString: ""
           pushNotifications:
             # provider - possible values: /"Dummy"/"Direct"
```

Comment:
that comment is now incorrect

+ you are missing enabled: true for the different providers.